Makefile

```diff
@@ -1,3 +1,10 @@
+ifeq ($(OS),Windows_NT)
+SEPARATOR := ;
+else
+SEPARATOR := :
+endif
+
+
```

Comment:
> Is this so that we can use a different separator per system? 

nailed it.  see https://github.com/Automattic/wp-desktop/pull/275/files#diff-b67911656ef5d18c4ae36cb6741b7965R1